api/python/quilt3/packages.py

```diff
@@ -318,6 +320,7 @@ def meta(self):
         return self._meta.get('user_meta', dict())
 
     @classmethod
+    @ApiTelemetry("Package.install")
```

Comment:
I feel like this should be outside of `@classmethod`? Though I'm not actually sure how `@classmethod` works.

Reply:
I believe a custom decorator should always be on the inside of a classmethod decorator - https://stackoverflow.com/questions/1987919/why-can-decorator-not-decorate-a-staticmethod-or-a-classmethod